src/main/resources/logback-spring.xml

```diff
@@ -21,39 +16,45 @@
     </springProfile>
 
     <springProfile name="!(dev || stdout)">
+        <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+            <layout class="ch.qos.logback.classic.PatternLayout">
+                <Pattern>${LOG_PATTERN}</Pattern>
+            </layout>
+        </appender>
         <appender name="logAppender" class="com.microsoft.applicationinsights.logback.ApplicationInsightsAppender"/>
     </springProfile>
 
```

Comment:
if you add
`    <logger name="uk.gov.justice.digital.hmpps.probationteams.ProbationTeamsApplication" additivity="false" level="DEBUG">
        <appender-ref ref="logAppender"/>
        <appender-ref ref="consoleAppender"/>
    </logger>`
it will show the application has start in the logs within the pod before they all go to app insights